wallet/createtx.go

```diff
@@ -416,11 +416,22 @@ func (w *Wallet) addrMgrWithChangeSource(dbtx walletdb.ReadWriteTx,
 // validateMsgTx verifies transaction input scripts for tx.  All previous output
 // scripts from outputs redeemed by the transaction, in the same order they are
 // spent, must be passed in the prevScripts slice.
-func validateMsgTx(tx *wire.MsgTx, prevScripts [][]byte, inputValues []btcutil.Amount) error {
-	hashCache := txscript.NewTxSigHashes(tx)
+func validateMsgTx(tx *wire.MsgTx, prevScripts [][]byte,
+	inputValues []btcutil.Amount) error {
```

Comment:
Nice that we already have this stuff available here, we somewhat worried how much extra plumbing we'd need to add everywhere to obtain all the additional prev out values.